particles/Tutorial/Javascript/README.md

```diff
@@ -113,26 +113,26 @@ particle ParentParticle in 'parent.js'
   // called "mySlot" in which another particle can render. The
   // child particle will be rendered inside a special div with the identifier
   // "mySlot", which this particle will need to provide in its HTML.
-  consume root
-    provide mySlot
+  root: consumes Slot
+    mySlot: provides
 
 // The "child" particle. Instead of consuming "root" it consumes "mySlot"
 particle ChildParticle in 'child.js'
-  consume mySlot
+  mySlot: consumes
 
 
 // Unlike previous recipes, this one includes two particles.
 recipe RenderSlotsRecipe
   ParentParticle
     // The ParentParticle consumes root just like particles in previous examples.
-    consume root
+    root: consumes
       // ParentParticle also provides mySlot. Note the additional tab over.
       // The name "slot" is the name the recipe uses for the slot, and is how
       // the recipe connects the parent's mySlot and the child's mySlot
-      provide mySlot as slot
+      mySlot: provides theSlot
   ChildParticle
     // And the ChildParticle consumes the slot.
-    consume mySlot as slot
+    mySlot: consumes theSlo
```

Comment:
`theSlo` here, but `theSlot` above -- maybe the name is being ignored?

Reply:
Fixed.